fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -1589,6 +1605,168 @@ private static QueryParameter parseChainedParameter(Class<?> resourceType, Strin
         return rootParameter;
     }
 
+    /**
+     * Transforms the passed string representing reverse chain search criteria, into
+     * an actual chain of QueryParameter objects. This method consumes strings of this form:
+     * <pre>
+     *      +-------------------------------------------------------------------+
+     *      |                                                                   |
+     *      V                                                                   |
+     * >>---+--- "_has:{referenced-by-resource-type}:{reference-parameter}:" ---+--- "{search-parameter}" ---><
+     * </pre>
```

Comment:
mentioning this as it's the Eclipse Formatter control (turns on/off) 

```
@formatter:off 
@formatter:on
```

Reply:
added annotations